scripts/ci/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
could this file should have a more descriptive name than just "docker-compose.yml"?

Reply:
Potentially, but maybe the `ci` directory name is enough?

Reply:
I'll rename to "actions-docker-compose.yml" since it's having to be used with the `-f` argument anyway and so `docker-compose`'s default of looking for `./docker-compose.yml` doesn't help.